autoscheduler/scraper/models/instructor.py

```diff
@@ -1,4 +1,10 @@
 from django.db import models
 
 class Instructor(models.Model):
-    """ Describe model here """
+    """ Model that represents a professor, who may teach many courses """
+    id = models.IntegerField(primary_key=True)
+    email_address = models.CharField(max_length=320, null=True)
+    name = models.CharField(max_length=64)

```

Comment:
Do you think it's necessary to include their [bannerId](https://github.com/gannonprudhomme/AutoScheduler/blob/40fb364b8d6181839a5badfffcbf0ea426276853/autoscheduler/scraper/tests/section_inputs.json#L35) as well?

Reply:
No, we only need one or the other, as far as I can tell

Reply:
I searched checked if there were any duplicate entries and it seems like both of them are unique regardless, so it really shouldn't matter which we use. Given that integers are probably always going to be faster that strings for searching/querying, I think sticking with the pidm is fine.